packages/vuetify/src/entry-lib.ts

```diff
@@ -1,4 +1,5 @@
+import * as directives from './directives'
 export * from './components'
-export * from './directives'
+export { directives }
```

Comment:
Why are you changing exports? It's just an incorrect typescript definition.

Reply:
according to the documentation and the other files it's expected that from vuetify/lib that the directives were supposed to be encapsulated in a 'directives' object.  I was trying to re-align it with what was here: https://github.com/vuetifyjs/vuetify/blob/64056b45fedb8781c9415812117923b0c83f505b/packages/vuetify/types/lib.d.ts 

If that's changed and the expectation is that the directives are individually exported just like the components, then the lib.d.ts is wrong and we'll have to update that as well.

Reply:
Yes, types and documentation are just documentation. If someone is successfully using this without typescript, then this change would be a breaking one.

Reply:
So which one is correct? the documentation and types or the entry-lib? From everything I've seen this is what's expected, but maybe it was changed and the docs and types are out of date?  I just want the types to align, I'm not opposed to changing either, but it just started to happen in a recent update is all.

Reply:
The source is correct.

Reply:
I've reverted this change back to what was in the lib as per the conversation.  Updated the typescript lib to match instead